validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
+# pylint: disable=inconsistent-return-statements
```

Comment:
Which functions does this apply to? Can it be moved to just those functions, instead of across the entire file?

Reply:
I can't figure out how to apply this to just the functions. I could leave the line numbers that it applies to in a comment. I thought that since we are re-writing this in Rust very soon, I shouldn't spend too much time on it.